src/Editor.tsx

```diff
@@ -335,6 +335,7 @@ class Editor extends Component<Props> {
         if (e.attributes) {
           e.attributes.color = undefined;
           e.attributes.background = undefined;
+		  e.attributes["code-block"] = false;
```

Comment:
Indentation in code blocks are two spaces. Just match what you have above!

Reply:
done!